src/Components/Web/src/PersistenceReasons.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Components.Web;
+
+/// <summary>
+/// Represents persistence during prerendering.
+/// </summary>
+public class PersistOnPrerendering : IPersistenceReason
+{
+    /// <inheritdoc />
+    public bool PersistByDefault { get; } = true;
+}
```

Comment:
Use the singleton pattern here and provide a static `Instance` value that gets reused everywhere